planetary-social/nos
# Send to nos account moderation

## Issues covered
Covers https://github.com/planetary-social/nos/issues/1083, https://github.com/planetary-social/nos/issues/1093, https://github.com/planetary-social/cleanstr/issues/44, https://github.com/planetary-social/nos/issues/1084
## Description

It adds a `Send To Nos` option in the account menu similar to the note version so that users can have flag anonymously

## How to test
1. Navigate to some user's menu
2. Select Flag this User
3. Select a category
4. Select `Send to Nos`
5. Confirm Dialog
6. Verify report in Slack channel
